tests/unit/composables/LetoModelizerApi.test.js

```
@@ -99,4 +100,26 @@ describe('User Authentication', () => {
       expect(res.results).toEqual(roles);
     });
   });
+
+  describe('Test function: manageError', () => {
+    it('should throw an error on not 503', async () => {
```

should throw an error when status is not 503

---

keep justine sentence